src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -157,6 +192,20 @@ export class FormSignInComponent implements OnInit {
     }
   }
 
+  oauthAuthorize(value: DeclareOauthSession) {
```

Comment:
This call might not be required since the call to the 

`this._router.navigate(['/oauth/authorize']`

might be enough, since it triggers the Guard that declares the section 

https://github.com/ORCID/orcid-angular/blob/8ca1a53ea559f25a69c39d4e4d6a6cbb0c13fb1f/src/app/guards/oauth.guard.ts#L35

But I will take a look at this later while finishing the oauth work

Reply:
Ok, let's take a look later to this